### PR TITLE
Feature/initial

### PR DESCRIPTION
**Pull request by lats1**

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+node_modules
+npm-debug.log
+.env
```
Comment (rimi-itk) [inline .dockerignore]: (Probably) Missing newline at end of file.

File: README.md [hunk 2]
```diff
@@ -1 +1,94 @@
...
+
+## Release
+
+This software does not follow a versioning scheme, instead the latest production ready version is the one in the master branch.
+
+Whenever changes is merged in to the master branch, a docker image should be build and pushed to the package repository:
+```bash
+docker build -t itkdev/borgerdata .
+docker tag <DOCKER_IMAGE> docker.pkg.github.com/itk-dev/borgerdata/app:latest
+ocker push docker.pkg.github.com/itk-dev/borgerdata/app:latest
```
Comment (rimi-itk) [inline README.md]: Should be `docker push …` – I think …

File: README.md [hunk 3]
```diff
@@ -1 +1,94 @@
...
+```
+
+## Deployment
+ 
+ You will need an environment where the following is present:
+
+- [Node js](https://nodejs.org) v10.17.0.
+- [Yarn](https://yarnpkg.com/) v1.19.1.
```
Comment (rimi-itk) [inline README.md]: Can we set these requirements in `package.json`?

File: migrations/20191206083531_initBorgerdata.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
+
+exports.up = async function(knex) {
```
Comment (rimi-itk) [inline migrations/20191206083531_initBorgerdata.js]: Is this table used only for test/development or are we caching data from the real SQL Server?

Comment (lats1) [reply migrations/20191206083531_initBorgerdata.js]: Yup, they are only for test/development. I've updated the README with a little bit of info about this

**Pull request merged**